Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM python:3.7
+ARG BASE=3.8
+FROM python:$BASE
```

Comment:
@tacaswell looks good. I'm used to just having the image _with_ tag as the whole build arg, but I think this will work but I will check and then confirm.

Reply:
Yeah, this works. My default is

```docker
ARG BUILDER_IMAGE=python:3.8
FROM ${BUILDER_IMAGE} as builder
```

to just make things very explicit (the `builder` stuff is just because I almost always use multistage builds but probably not needed here now). But I have confirmed that

```
docker build . \
       -f Dockerfile \
       --build-arg BASE=3.8 \
       --tag matplotlib/mpl-docker:debug-3.8
```

works.

Reply:
That makes sense.